docker-compose.yml

```diff
@@ -177,7 +177,7 @@ services:
       - POSTGRES_PASSWORD=openapparelregistry
       - POSTGRES_DB=openapparelregistry
       - GIT_COMMIT=Unknown
-    command: uvicorn app.main:app --host 0.0.0.0 --port 84 --reload --workers 8
+    command: uvicorn app.main:app --host 0.0.0.0 --port 84 --workers 1
```

Comment:
What do you think, about checking the environment and leaving `--reload` only for the development env if it could be useful?

Reply:
@mazursasha1990  I didn't think so, as with current setup this option didn't work properly even on local env.